tensorflow/python/module/module.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, input_size, sizes, name=None):
       self.layers = []
       with self.name_scope:
         for size in sizes:
-          self.layers.append(Dense(input_size=input_size, output_size=size))
+          self.layers.append(Dense(input_dim=input_size, output_size=size))
```

Comment:
Maybe convert this to a doctest like the above Dense class so that errors like these are caught in tests?

Reply:
I see, any other changes you would recommend?

Reply:
I can't think of any other changes yet.

Reply:
I will put it in doctest form then. If anything else comes to mind I will do that too.

Reply:
Thank you!

Reply:
I added the test as requested. While doing that I noticed that the doctest for the Dense class example has a related issue with param names so I did the same thing there. 

I'm going to attach notebook with results output of adjusted code.

Reply:
Thanks!